packages/protocol-kit/src/types/safeProvider.ts

```diff
@@ -11,12 +14,12 @@ export type HexAddress = string
 export type PrivateKey = string
 export type HttpTransport = string
 export type SocketTransport = string
-export type SafeSigner = HexAddress | PrivateKey
+export type SafeSigner = HexAddress | PrivateKey | PasskeySigner
 
 export type SafeProviderConfig = {
   /** signerOrProvider - Ethers signer or provider */
   provider: Eip1193Provider | HttpTransport | SocketTransport
-  signer?: HexAddress | PrivateKey
+  signer?: HexAddress | PrivateKey | passkeyArgType
```

Comment:
Isnt this the `SafeSigner` above?

Reply:
nope, in the `SafeSigner` the `PasskeySigner` is a different type